CONTRIBUTING.adoc

```diff
@@ -0,0 +1,240 @@
+[[contributing-to-omni-core]]
+= Contributing to Omni Core
+DexX7
```

Comment:
Do you think we should state author (or version) here at all? I'd assume the commit history is usually more accurate, and it may be a hassle to keep this updated.